demo/magic_8_ball/run.py

```diff
@@ -0,0 +1,135 @@
+import io
```

Comment:
<img width="1257" alt="image" src="https://github.com/user-attachments/assets/a8241773-a67d-4d7d-bc18-aca652bbd887">

Sweet! I asked it what career I should choose

Btw the generation takes quite long -- can we cache some examples so that users can get an idea of what it sounds like?